package.json

```diff
@@ -11,6 +11,9 @@
   "scripts": {
     "build": "vite build",
     "dev": "vite",
+    "generate-pwa-assets": "npx pwa-asset-generator ./public/images/icon-vector.svg ./pwa-assets -m ./public/manifest.json -i ./index.html --sizes \"72x72,96x96,128x128,144x144,152x152,192x192,384x384,512x512\" --quality 75 --background \"#ffffff\" --padding \"10%\" --platform android,ios --opaque false",
+    "move-pwa-assets": "rm -rf ./public/pwa-assets && mv ./pwa-assets ./public/",
+    "pwa-setup": "npm run generate-pwa-assets && npm run move-pwa-assets",
```

Comment:
1. "generate-pwa-assets":
   - Takes our vector icon from public/images/icon-vector.svg
   - Generates various sizes of icons required for PWA (72px to 512px)
   - Updates manifest.json and index.html with the new icons
   - Applies settings:
     * 75% image quality for optimization
     * White background
     * 10% padding around icons
     * Compatible with both Android and iOS
     * Transparent background allowed

2. "move-pwa-assets":
   - Moves generated assets to the public directory
   - Cleans up old assets first to avoid duplicates

3. "pwa-setup":
   - Runs both commands in sequence to generate and organize PWA assets